netbox/netbox/models/features.py

```diff
@@ -227,6 +231,8 @@ def get_custom_fields_by_group(self):
 
         for cf in visible_custom_fields:
             value = self.custom_field_data.get(cf.name)
+            if not value and cf.ui_visibility == CustomFieldVisibilityChoices.VISIBILITY_HIDDEN_IFUNSET:
```

Comment:
this hides boolean custom fields where the value is set to false.

this works and shows boolean fields that are set to false
``` python
if value == None and cf.ui_visibility == CustomFieldVisibilityChoices.VISIBILITY_HIDDEN_IFUNSET:
```